greenheart/tools/eco/electrolysis.py

```diff
@@ -33,37 +32,43 @@ def run_electrolyzer_physics(
     electrolyzer_size_mw = greenheart_config["electrolyzer"]["rating"]
     electrolyzer_capex_kw = greenheart_config["electrolyzer"]["electrolyzer_capex"]
     
+    # IF GRID CONNECTED
     if greenheart_config["project_parameters"]["grid_connection"]:
-        energy_to_electrolyzer_kw = np.ones(365 * 24 - 4*7*12) * ( # TODO why the subtraction here?
-            electrolyzer_size_mw * 1e3
-        )
+        n_pem_clusters = 1
+        grid_connection_scenario='grid-only'
+        hydrogen_production_capacity_required_kgphr=greenheart_config["electrolyzer"]["sizing"]["hydrogen_dmd"]
+        energy_to_electrolyzer_kw = []
+        
+            
+
+
+    # IF NOT GRID CONNECTED
     else:
+        hydrogen_production_capacity_required_kgphr = []
+        grid_connection_scenario = 'off-grid'
         energy_to_electrolyzer_kw = np.asarray(hopp_results[
             "combined_hybrid_power_production_hopp"
         ])
-    # calculate utilization rate
-    energy_capacity = electrolyzer_size_mw * 365 * 24  # MWh
-    energy_available = sum(energy_to_electrolyzer_kw) * 1e-3  # MWh
-    capacity_factor_electrolyzer = energy_available / energy_capacity
-
+        n_pem_clusters = electrolyzer_size_mw//greenheart_config["electrolyzer"]["cluster_rating_MW"]
+        
     ## run using greensteel model
-    pem_param_dict = {"Modify EOL Degradation Value": False,
-                      "EOL Rated Efficiency Drop": 0.1,
+    pem_param_dict = {"Modify EOL Degradation Value": greenheart_config["electrolyzer"]["custom_EOL_efficiency_drop"],
+                      "EOL Rated Efficiency Drop": greenheart_config["electrolyzer"]["EOL_efficiency_drop"],
                       "Modify BOL Eff": False,
-                      "BOL Eff [kWh/kg-H2]": 0.95}
+                      "BOL Eff [kWh/kg-H2]": []}
```

Comment:
This is because it was a capability I tried to add and wasn't able to figure out. It's unused and honestly - this would be a good time for me to remove it entirely.